gcp/appengine/frontend_handlers.py

```diff
@@ -300,7 +300,15 @@ def add_source_info(bug, response):
   response['source'] = source_repo.link + source_path
   response['source_link'] = response['source']
   if source_repo.human_link:
-    response['human_source_link'] = source_repo.human_link + bug.id()
+    base_ecosystem = bug.ecosystem[0]
+    bug_id = bug.id()
+    if base_ecosystem == 'AlmaLinux':
+      base_ecosystem = bug.ecosystem[1].split(':')[1]
+      bug_id = bug_id.replace(':', '-', 1)
+    if base_ecosystem == 'GIT' and bug_id.startswith('CURL-'):
+      bug_id = bug_id.removeprefix('CURL-')
+    response['human_source_link'] = render_template_string(
+        source_repo.human_link, BASE_ECOSYSTEM=base_ecosystem, BUG_ID=bug_id)
```

Comment:
I think there's a happy medium here for us to use flask/jinja filters here for the very ecosystem-specific bits (e.g. https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.replace).

This way we don't have ecosystem-specific code here, and we don't expose internal Bug entity details in the yaml.

Reply:
To be clear about what I mean here, this would look something like:

```
{{ BUG_ID | replace("CURL-", "") }}
```

etc

Reply:
Got it. Addressed in f1239d5